airbrake/notifier.py

```diff
@@ -145,6 +145,12 @@ def __init__(self, project_id=None, api_key=None, host=None, timeout=None,
 
         self._exc_queue = utils.CheckableQueue()
 
+        self._session = requests.Session()
```

Comment:
It's understandable why we'd want to reuse the TCP connection but I'd expect to see an explanation in the commit message. Would you mind amending the commit explaining why a session object was added?